tests/org.eclipse.papyrus.uml.interaction.model.tests/src-gen/org/eclipse/papyrus/uml/interaction/model/tests/MLifelineTest.java

```diff
@@ -733,20 +806,4 @@ public void testMakeCreatedAt__OptionalInt() {
 		assertThat("Center line not moved", getFixture().getTop(), isPresent(expectedTop.getAsInt()));
 	}
 
-	public void testMakeCreatedAt__OptionalInt__empty() {
```

Comment:
How did this get deleted?  It didn't have an `@generated not` tag.  Please restore it.

Reply:
Restored in commit 348a8d6, probably a merge conflicts resolution failure on my side.